.github/workflows/builder-release.yaml

```diff
@@ -5,8 +5,13 @@ on:
     tags:
       - 'cmd/builder/v*'
 
+permissions:
+  contents: read
+
 jobs:
   goreleaser:
+    permissions:
+      contents: write  # for goreleaser/goreleaser-action to create a GitHub release
```

Comment:
ditto here (well, here it would not take a long time to notice but it would affect the release, so I would prefer to have this tested first)